requirements.txt

```diff
@@ -1,4 +1,5 @@
 hug==2.5.6
-maxminddb==2.0.2
+maxminddb==2.2.0
 uwsgi==2.0.18
 awscli==1.16.180
+geoip2==3.0.0
```

Comment:
why this version? This was released in Dec 21, 2019.
The latest is 4.6.0.
https://pypi.org/project/geoip2/